tedana/utils.py

```diff
@@ -208,7 +209,21 @@ def dice(arr1, arr2, axis=None):
         dsi = np.zeros(arr_sum.shape)
     else:
         intersection = np.logical_and(arr1, arr2)
-        dsi = (2.0 * intersection.sum(axis=axis)) / arr_sum
+        # Count number of zero-elements in the denominator and report
+        total_zeros = np.count_nonzero(arr_sum == 0)
+        if total_zeros > 0:
+            LGR.warning(
+                f"Calculating dice coefficient with {total_zeros} "
+                "zero-elements in the denominator. "
+                "Please check your component table for dice columns with 0-"
+                "values"
```

Comment:
How about something like `f"{total_zeros}/{arr_sum.size} components have empty maps, resulting in Dice values of 0. Please check your..."`